### DISCUSSION TITLE
i will start another project  viewer of age

### DISCUSSION THREAD

**Discussion by shinhanbyeol** [apache/age#1780] [Show and tell]
Hello, age contributors

I haven't contributed much to the "age-viewer" for a long time due to personal things, but after working on other projects and things in the meantime, I wanted to take on a new challenge. 

With the help of many people, "age-viewer" is maintained, but I want to develop it with "typescript" rather than "javascript" and I want to develop it in a different form than before. 

 i will use "electron" and "Next.js" as well as "sqlite".

First of all, I'm going to open a separate project in my personal repository and proceed with it, and anyone who is interested or wants to give me an idea is welcome.

We will be working on it publicly [here](https://github.com/shinhanbyeol/age-viewer-2.0) in the repository, and we will let you know the good news again when the main features are developed soon.

**Comment by markgomer** [comment]
I personally think Apache AGE would be much more popular if AGE-Viewer had more attention, thanks for this initiative! I'm eager to contribute to it anyway I can!

**Reply by mingfang** [reply]
I agree 1M%
That's one of the reason why Neo4j is so popular.

I created this "viewer" as a way to learn AGE.
https://age-playground.rebelsoft.com/

**Reply by shinhanbyeol** [reply]
Thank you for your interest.

Not much has been done yet, so I'll update you again when I implement the main features of the existing "age-viewer" similarly! 

If you have an idea or a code you want to contribute, it would be nice to do a discussion or pull request on the repo I shared.

**Reply by HackettJP** [reply]
thks for showing your viewer, very help full in understanding under the hood , and how AGE works

**Comment by aked21** [comment]
Hello @shinhanbyeol

Thanks for the suggestion!
Can you look into the following AGE viewer branch (under Apache AGE) to see if you can contribute to there? 
[AGE Viewer Golang ](https://github.com/apache/age/tree/ageviewer_go)

Thanks

**Reply by shinhanbyeol** [reply]
I will also refer to the code.

It would be an honor to be able to donate to this project Apache later if other committer and community members agree on the assumption that this project is successful.

**Comment by dpdjvhxm** [comment]
@shinhanbyeol @markgomer 

I agree that AGE Viewer will be help, AGE.

I'm interest to participate AGE Viewer develop, currently I'm contributing Apache AGE Website. 

I will check your repo

Thanks
